### PR TITLE
Update I18n.php

### PR DESCRIPTION
**Pull request by WinterSilence**

### CODE REVIEW

File: system/classes/Kohana/I18n.php [hunk 1]
```diff
@@ -124,43 +141,31 @@ public static function load($lang)
...
 	 *
 	 * @uses    I18n::get
 	 * @param   string  $string text to translate
 	 * @param   array   $values values to replace in the translated text
-	 * @param   string  $lang   source language
+	 * @param   string  $lang   target language
 	 * @return  string
 	 */
-	function __($string, array $values = NULL, $lang = 'en-us')
+	function __($string, array $values = NULL, $lang = NULL)
```
Comment (piotrbaczek) [inline system/classes/Kohana/I18n.php]: Why did you remove default lang?

Comment (WinterSilence) [reply system/classes/Kohana/I18n.php]: i dont do this https://github.com/koseven/koseven/pull/292/commits/b281407660117983edbb8bec6e7fe7f614653290#diff-83f0de0a4344b84921a83df373244907R87

Comment (WinterSilence) [reply system/classes/Kohana/I18n.php]: @piotrbaczek think, what you need to do to change default language before and now? :)

**Comment by toitzi** [comment]
Closed Because of inactivity. Issue is still opened but needs detailed review.

**Comment by toitzi** [comment]
Restored and Reopend. 
Explanation: https://github.com/koseven/koseven/pull/290#issuecomment-453875022

**Comment by toitzi** [comment]
Fixed with #321
